gridt/src/app/settings/settings.page.html

```diff
@@ -24,11 +24,9 @@
     <ion-row justify-content-center>
       <ion-col size="12" size-sm="8" offset-sm="2" size-md="6" offset-md="3" text-center>
         <ion-card>
-
-          <!-- <img src="https://www.gravatar.com/avatar/HASH"> -->
           <ion-card-header>
-            <img src="https://www.gravatar.com/avatar/HASH">
-            <ion-card-title>John Doe</ion-card-title>
+            <img [src]="(identity$ | async)?.gravatar">
```

Comment:
The gravatar will not be accessible in this manner. (This was a recent update on the server side.) Instead call md5 on identity.email. (With the appropriate async stuff of course.)

Reply:
You might need to install a library for this or copy the code from css-tricks.